notebooks/115-async-api/115-async-api.ipynb

```diff
@@ -324,28 +320,32 @@
     "        # Start capturing\n",
     "        start_time = time.time()\n",
     "        player.start()\n",
-    "        # Read CURRENT frame\n",
-    "        frame = player.next()\n",
     "        if use_popup:\n",
     "            title = \"Press ESC to Exit\"\n",
     "            cv2.namedWindow(title, cv2.WINDOW_GUI_NORMAL | cv2.WINDOW_AUTOSIZE)\n",
+    "        # Capture CURRENT frame\n",
+    "        frame = player.next()\n",
+    "        resized_frame = preprocess(frame)\n",
+    "        curr_request.set_tensor(input_layer_ir, ov.Tensor(resized_frame))\n",
+    "        # Start the CURRENT inference request\n",
+    "        curr_request.start_async()\n",
     "        while True:\n",
-    "            # Read NEXT frame\n",
+    "            # Capture NEXT frame\n",
     "            next_frame = player.next()\n",
-    "            if frame is None:\n",
+    "            if next_frame is None:\n",
```

Comment:
You can leave the name "frame" instead of the "next_frame" everywhere you use it. It would make code easier to understand.

Reply:
Hi I dont think so, next_frame and frame are for 2 infer requests separately

Reply:
Hi I dont think so, next_frame and frame are for 2 infer requests separately

Reply:
You're only assigning the reference to the variable, you're not changing the value itself. Try it :)